### DISCUSSION TITLE
Possible bug - Uno Templates - cannot launch app if app name has spaces.

### DISCUSSION THREAD

**Discussion by rruprai1** [unoplatform/uno#18253] [Q&A]
Could be a known bug, (mabye withs vscode?)

Have an app name with spaces for example (exclude the quotes) "My foo - bar" . Use VSCode on MacOS to build the application using a launch configuration similar to:
{
      "type": "Uno",
      "request": "launch",
      "preLaunchTask": "Uno: net8.0-maccatalyst | Debug | maccatalyst-x64",
 }
 
Result:
The application will not launch. 

from what I see the path to the app is mangled, the launch will fail with exit code 1 and the path used would be something like:
"bin/maccatalyst/AnyCPU/Debug/My, /foo, /-, /bar.app" 
which results in exit code 1 and the message 'the files do not exist"

Note, everything builds fine, the folders and build output will exist

**Comment by jeromelaban** [comment]
Thanks for the report! This is a known issue, for which we have no control over as it's managed by the .NET templates which don't support well that scenario. We prevent that scenario when using the live wizard, though: https://new.platform.uno.
